### PR TITLE
Removed uninstall.ps1, updated to 2.1

### PR DESCRIPTION
**Pull request by Roemer**

### CODE REVIEW

**Comment by Roemer** [comment]
Might fix issue #5  and issue #8 and issue #6.

File: msbuild-sonarqube-runner/tools/chocolateyinstall.ps1 [hunk 1]
```diff
@@ -1,8 +1,5 @@
...
-Install-ChocolateyZipPackage 'msbuild-sonarqube-runner' `
-  -url "$url" `
-  -unzipLocation "$(Split-Path -parent $MyInvocation.MyCommand.Definition)" `
-  -checksumType 'sha1' `
-  -checksum "$checksum"
+﻿$packageName = 'msbuild-sonarqube-runner'
+$url = 'https://github.com/SonarSource-VisualStudio/sonar-msbuild-runner/releases/download/2.1/MSBuild.SonarQube.Runner-2.1.zip'
+$url64 = $url
+$toolsDir = "$(Split-Path -parent $MyInvocation.MyCommand.Definition)"
+Install-ChocolateyZipPackage $packageName $url $toolsDir $url64
```
Comment (ghost) [inline msbuild-sonarqube-runner/tools/chocolateyinstall.ps1]: Can you add back the checksum? It's [highly recommended](https://github.com/chocolatey/choco/wiki/HelpersInstallChocolateyZipPackage#-checksum-string) to have it.

File: msbuild-sonarqube-runner/msbuild-sonarqube-runner.nuspec [hunk 2]
```diff
@@ -3,13 +3,15 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>msbuild-sonarqube-runner</id>
-    <title>SonarQube Scanner for MSBuild</title>
-    <version>1.1.0.20160205</version>
+    <title>SonarQube Runner for MSBuild</title>
```
Comment (ghost) [inline msbuild-sonarqube-runner/msbuild-sonarqube-runner.nuspec]: Although the executable and the github repo is both uses "runner", the product name is "SonarQube Scanner for MSBuild", so we should keep that.

File: msbuild-sonarqube-runner/msbuild-sonarqube-runner.nuspec [hunk 3]
```diff
@@ -3,13 +3,15 @@
...
+    <version>2.1</version>
     <authors>SonarSource,Microsoft</authors>
     <owners>SonarSource</owners>
-    <summary>The SonarQube Scanner for MSBuild allows easy analysis of any .NET project with SonarQube.</summary>
-    <description>The SonarQube Scanner for MSBuild allows easy analysis of any .NET project with SonarQube.</description>
+    <summary>The SonarQube Runner for MSBuild allows easy analysis of any .NET project with SonarQube.</summary>
+    <description>The SonarQube Runner for MSBuild allows easy analysis of any .NET project with SonarQube.</description>
     <projectUrl>http://redirect.sonarsource.com/doc/msbuild-sq-runner.html</projectUrl>
+	<projectSourceUrl>https://github.com/SonarSource-VisualStudio/sonar-msbuild-runner</projectSourceUrl>
```
Comment (ghost) [inline msbuild-sonarqube-runner/msbuild-sonarqube-runner.nuspec]: The tabulation is wrong here.

**Comment by ghost** [comment]
@Roemer Thank you for this PR. I added some minor inline comments. Could you please fix those before I merge to master.

**Comment by Roemer** [comment]
I added a new changeset with your feedback.

**Comment by ghost** [comment]
Thanks for the update @Roemer. It looks good to me.

**Comment by ghost** [comment]
I'll publish it to chocolatey.org.

**Pull request merged**